### DISCUSSION TITLE
How make a cage?

### DISCUSSION THREAD

**Discussion by nedcrow** [pmndrs/use-cannon#76] [General] [answered]
I'm made some cage. this cage was made in boxes.

But my wanted cage = one box.

This is possible?

Or can i reverse Box's collision?  

example, can i make cage and not used plane at next case

https://codesandbox.io/s/r3f-cannon-instanced-physics-devf8?file=/src/index.js

**Comment by codynova** [comment, marked as answer]
You could use `useCompoundBody` to join all the individual boxes into one compound shape, check out [the example here](https://github.com/react-spring/use-cannon/blob/master/examples/src/demos/CompoundBody.js)

**Reply by nedcrow** [reply]
Thank you i'll try!
